### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by st58594**
Jakub Pokorný, st58594

### CODE REVIEW

File: NNPTPZ1/Mathematics/ComplexNumber.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        {
+            RealPart = 0,
+            ImaginaryPart = 0
+        };
+
+        public double GetAbs()
+        {
+            return Math.Sqrt(RealPart * RealPart + ImaginaryPart * ImaginaryPart);
+        }
+        public double GetAngleInDegrees()
```
Comment (rodi0878) [inline NNPTPZ1/Mathematics/ComplexNumber.cs]: Really degrees?

File: NNPTPZ1/Mathematics/ComplexNumber.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                return complexNumber.RealPart == RealPart && complexNumber.ImaginaryPart == ImaginaryPart;
+            }
+            return base.Equals(objectToCompare);
+        }
+
+        public override string ToString()
+        {
+            return $"({RealPart} + {ImaginaryPart}i)";
+        }
+        internal ComplexNumber Divide(ComplexNumber b)
```
Comment (rodi0878) [inline NNPTPZ1/Mathematics/ComplexNumber.cs]: Please reorder class and move all math operations to one part, Divide is separated by ToString and Equals...

File: NNPTPZ1/Mathematics/Polynomial.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        }
+
+        /// <summary>
+        /// ToString
+        /// </summary>
+        /// <returns>String representation of polynomial</returns>
+        public override string ToString()
+        {
+            string result = "";
+            int i = 0;
```
Comment (rodi0878) [inline NNPTPZ1/Mathematics/Polynomial.cs]: why declare and initialize i here?

File: NNPTPZ1/NewtonFractal.cs [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+                new List<ComplexNumber> {
+                    new ComplexNumber{ RealPart = 1},
+                    ComplexNumber.Zero,
+                    ComplexNumber.Zero,
+                    new ComplexNumber{ RealPart = 1},
+                }
+             );
+        }
+
+        private int FindRoot(ComplexNumber pixel)
```
Comment (rodi0878) [inline NNPTPZ1/NewtonFractal.cs]: Name of method is misleading

**Review by rodi0878 — CHANGES_REQUESTED** [review]
please see comments and provide update

**Comment by st58594** [comment]
Thank you for the feedback. Updated.

**Comment by rodi0878** [comment]
accepted

**Pull request closed without merging**